### ISSUE TITLE
another special char problem

### ISSUE THREAD

**Issue by wotanzheng** [ssut/py-googletrans#14]
It's me again. :)

I found there is another problem with special char - Guillemet (《). When an Chinese or Japanese char is behind a Guillemet, the program will get an error just like #13

**Comment by wotanzheng** [comment]
for example:《财

**Comment by wotanzheng** [comment]
》has the same error. I think you can review special char list. :)

**Comment by ssut** [comment]
Jsut for the memo here, the deobfuscated code for google translate token generator:

```javascript
var a = "©×";
for (var result = [], counter = 0, i = 0; i < a.length; i++) {
    var l = a.charCodeAt(i);
    if (128 > l) {
        result[counter++] = l
    } else {
        if (2048 > l) {
            result[counter++] = l >> 6 | 192
        } else {
            if (55296 == (l & 64512) && i + 1 < a.length && 56320 == (a.charCodeAt(i + 1) & 64512)) {
                l = 65536 + ((l & 1023) << 10) + (a.charCodeAt(++i) & 1023)
                result[counter++] = l >> 18 | 240
                result[counter++] = l >> 12 & 63 | 128
            } else {
                result[counter++] = l >> 12 | 224
                result[counter++] = l >> 6 & 63 | 128
            }
        }
        
        result[counter++] = l & 63 | 128
    }
}

console.log(result);
```

**Comment by ssut** [comment]
Solved this issue with a new release 2.1.2.
